src/circular-progress.jsx

```diff
@@ -72,15 +72,17 @@ const CircularProgress = React.createClass({
     this._rotateWrapper(wrapper);
   },
 
+  componentWillUnmount() {
+    clearTimeout(this._scalePathTimer);
```

Comment:
well you have a point there. Alright I'm sold. Do you want me to remove `_` from your code too?